app/javascript/app/components/slide-cards/slide-cards-component.jsx

```diff
@@ -63,9 +63,9 @@ class SlideCards extends Component {
                   contentFirst
                   title={{
                     link: card.source.link,
-                    title: `Targets ${card.source.title !== ''
-                      ? `found in ${card.source.title}`
-                      : ''}`
+                    title: `${card.source.title !== ''
+                      ? `Targets found in ${card.source.title}`
+                      : 'Targets'}`
```

Comment:
you don't need to check if it's empty like this: ${card.source.title !== '' 
empty string is falsy in js so you can shortify it to: card.source.title ? `Targets found in .....` : 'Targets'